ci/buildspec.yml

```diff
@@ -28,8 +28,9 @@ phases:
       # integration test is broken: https://github.com/awslabs/mxnet-model-server/issues/437
       #- python -m pytest mms/tests/integration_tests
       - pylint -rn --rcfile=./mms/tests/pylintrc mms/.
+      - python setup.py bdist_wheel --universal
```

Comment:
Can we remove unnecessary dependencies?
1.  apt-get install -y libgfortran3?
2. pip install flask
3. pip install flask-cors
4. pip install requests
5. pip install jsonschema
6. pip install -U -e . ?